server/datastore/mysql/hosts.go

```diff
@@ -1962,6 +1962,50 @@ WHERE uuid IN (?) AND %s
 	return hosts, nil
 }
 
+func (ds *Datastore) ListHostsLiteByIDs(ctx context.Context, ids []uint) ([]*fleet.Host, error) {
```

Comment:
Should we document that not all fields of the host are returned? Maybe define a new `HostLite` struct?

Reply:
I was reusing the `HostLite` name on purpose in the method name, as that's a concept that already exists. I agree that it would be nice to have this translated to the type system in some way, but the thing is that there are a lot of stuff associated with the main `Host` struct (authz, `DisplayName` method in this case, etc.) which would make this a relatively involved change. The best I can think if is to wrap `Host` in a new struct but that would just be an indication via the new struct name, the unfilled fields would still be accessible, not sure that's worth it (ie:

```
type HostLite struct {
    *Host
}
```

Reply:
Am ok with documenting what fields are exactly returned everywhere we define/declare `ListHostsLiteByIDs`.

Reply:
:+1: yeah that makes sense, I'll just document that it returns a subset and to check the implementation for the exact list of fields, otherwise it's too high a risk for outdated comments as soon as we change that list of fields.